m0t1fs/linux_kernel/file.c

```diff
@@ -2317,70 +2317,80 @@ static int pargrp_iomap_paritybufs_alloc(struct pargrp_iomap *map)
 			   "data_buf.", map->pi_ioreq);
 }
 
-static int pargrp_iomap_populate(struct pargrp_iomap        *map,
-				 struct m0_ivec_varr_cursor *cursor)
+/**
+ * Set segment by countiguous user data from cursor until the
+ * parity group end.
```

Comment:
* Set segment by countiguous user data from cursor at most to the
 * parity group end. If the contiguous user data is less than the group end,
 * the result segment is only to the end of that contiguous region.

Reply:
Ok, will update.